src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -210,6 +210,7 @@ def test_aks_create_aadv1_and_update_with_managed_aad(self, resource_group, reso
                      '--aad-server-app-secret fake-secret ' \
                      '--aad-client-app-id 00000000-0000-0000-0000-000000000002 ' \
                      '--aad-tenant-id d5b55040-0c14-48cc-a028-91457fc190d9 ' \
+                     '--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/AADv1AllowCreate ' \
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79116982&view=results) to validate the change.

Reply:
Test passed.